Source/View+SnapKit.swift

```diff
@@ -178,6 +178,58 @@ public extension View {
             objc_setAssociatedObject(self, &installedLayoutConstraintsKey, newValue, objc_AssociationPolicy.OBJC_ASSOCIATION_RETAIN_NONATOMIC)
         }
     }
+    
+    internal var snp_constraints: [Constraint] {
```

Comment:
I'm afraid `snp_layoutConstraints` doesn't work in this case because we need to store the `Constraint` objects that will create the `LayoutConstraints` for some size classes. If the `LayoutConstraint` doesn't exist yet then there is no way we can reverse it and obtain our `Constraint`. Also we need these constraints stored in `toInstallView` instead of `fromView`.